### PR TITLE
Style (spam) bulk removal for moderators

### PR DESCRIPTION
**Pull request by a0eoc**

### CODE REVIEW

File: web/views/email/style/bulkban.html.tmpl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<p>
+	Styles that were removed:
+	<ul>
+	{{ range .Styles -}}
+		<li>
+			{{ .Name }}
+		</li>
+	{{ end }}
+	</ul>
```
Comment (vednoc) [inline web/views/email/style/bulkban.html.tmpl:18]: Why is this list inside of a `p` tag?

File: web/views/email/style/bulkban.text.tmpl [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Some of your styles have been removed from our platform for the following reason:
+{{ .Log.Reason }}
+
+Styles that were removed:
+{{ range .Styles -}}
+- {{ .Name }}
+{{ end }}
+
+{{ with .Log.Message }}	Additional message from the moderator: {{ . }}{{ end }}
```
Comment (vednoc) [inline web/views/email/style/bulkban.text.tmpl:13]: Is the tab here intentional?

File: web/views/email/style/bulkban.html.tmpl (hunk 1 above)
Comment (a0eoc) [reply web/views/email/style/bulkban.html.tmpl:18]: To somewhat separate it from other parts and for the consistency as now all the parts are kept inside such tags.  
I don't really know HTML and how to write in it, nor have I tested how this email would look like rendered yet.  
Wrongness of these `p`s was already discussed a bit when we were moving these email templates to files but I still don't know how that should be done.

File: web/views/email/style/bulkban.text.tmpl (hunk 2 above)
Comment (a0eoc) [reply web/views/email/style/bulkban.text.tmpl:13]: Nope, it migrated from the message for the regular style removal email like that when I was trying to get rid of those empty lines in the rendered code. I didn't have highlight for wrong tab/spaces enabled in the editor. Will fix that later.

**Review by vednoc — APPROVED** [review]

**Pull request merged**